### PR TITLE
Implementation of temperature based losses

### LINKED ISSUE #6: Consider losses due to heating and cooling of EVs
Losses due to heating and cooling are currently not part of simBEV.

- In a first version, this could be achieved using a single temperature timeseries for Germany.
- Later, more detailed data (e.g. on weather cells of [ERA5](https://www.ecmwf.int/en/forecasts/datasets/reanalysis-datasets/era5)) could be used. Simulated regions from CSV need to have a weather cell assigned.

Some tasks, feel free to amend:
- [ ] Find robust loss params
  - [this paper](https://arxiv.org/pdf/1801.05290.pdf) uses data from Tesla Model S (heating: 0.98%/°C, cooling: 0.63%/°C)
- [ ] Implement simple version with single temperature timeseries
- [ ] Use temperature timeseries on higher spatial resolution

### PR DESCRIPTION
**Pull request by mosc5**
This closes #6

### CODE REVIEW

File: simbev/simbevMiD.py [hunk 1]
```diff
@@ -1141,3 +1146,109 @@ def fast_charging_capacity(
...
+    start_date: str Start date of simulation
+    end_date: str End date of simulation
+    return: temperature_list list with different temperatures
+    """
+    # read in csv
+    temperature = pd.read_csv("data/Wetterdaten_Berlin.csv", sep=";", skiprows=1)
+    temperature = temperature[['date', 'tavg']]
+
+    # change format of start/end_date to same format as csv
+    [year, month, day] = start_date.split('-')
```
Comment (mosc5) [inline simbev/simbevMiD.py:1191]: anything from this point onward could be replaced if for every trip the timestep or date where it starts gets matched with a row/value of the weather pandas.

File: simbev/simbevMiD.py [hunk 2]
```diff
@@ -1141,3 +1146,109 @@ def fast_charging_capacity(
...
+
+    temperature: float the temperature for the timestep
+    timesteps: int timesteps for the trip
+    temperature_carinside: int
+    con: float consumption of the car
+    distance: the distance of the drive
+    return: consumption
+    """
+    energy_use_cooling = 0.01575
+    energy_use_heating = 0.025
```
Comment (nesnoj) [inline simbev/simbevMiD.py:1167]: Please add the source for those numbers

File: simbev/simbevMiD.py [hunk 3]
```diff
@@ -329,7 +332,8 @@ def availability(
                 staytime = math.ceil(staytime / stepsize)
                 # print("staytime done: " + str(staytime))
 
-                driveconsumption = distance * con
+                # driveconsumption = total_consumption(temperature_now, drivetime, temp_inside, cooling, heating, con,
+                #                                     distance)
```
[the remark this answers is not in the document]
Comment (mosc5) [reply simbev/simbevMiD.py:336]: gets set later (row 479)

**Comment by mosc5** [comment]
This has to be completely reworked to fit with the newest simbev version.

**Pull request closed without merging**